mock/cl0wihlop3rwc0mtj9np28ti2/get_asset_memoized/joblib/utils/helpers/get_asset_memoized/func_code.py

```diff
@@ -0,0 +1,27 @@
+# first line: 129
+@kili_project_memoizer(sub_dir="get_asset_memoized")
```

Comment:
I do not think that's the appropriate solution, let's do a real mocking. There are too many files created, and the binary storage is not readable. Happy to discuss about it.